src/main/java/com/recurly/v3/Client.java

```diff
@@ -223,28 +234,29 @@ public void removeBillingInfo(String accountId) {
   }
 
   /**
-   * @see <a href="https://partner-docs.recurly.com/v2018-08-09#operation/list_account_coupon_redemptions">list_account_coupon_redemptions api documentation</a>
-   *
-   * The QueryParam options available for this method are:
-   *   - "ids"
-   *   - "sort"
-   *   - "begin_time"
-   *   - "end_time"
-   *
+   * @see <a
+   *     href="https://partner-docs.recurly.com/v2018-08-09#operation/list_account_coupon_redemptions">list_account_coupon_redemptions
+   *     api documentation</a>
+   *     <p>The QueryParam options available for this method are: - "ids" - "sort" - "begin_time" -
```

Comment:
Could you upstream some of these changes to the generator so it's a little closer to 1-1 representation?